lib/config/resolvers/search.ts

```diff
@@ -1,7 +1,7 @@
 import { FacetFieldEnum } from "@/lib/graphql/generated/fbi/graphql"
 
 const search = {
-  "search.item.limit": 9,
+  "search.item.limit": 18,
```

Comment:
9 means faster load times also serverside. Could be nice to keep. Maybe 12 could be the sweetspot

```suggestion
  "search.item.limit": 9,
```

Reply:
If the results are non-divideable by both 2 and 3 then we end up with something like this:
![image](https://github.com/user-attachments/assets/3171fa01-a43e-4569-b344-5e7c77a780fd)